controllers/controller.js

```diff
@@ -1,8 +1,12 @@
-const { fetchTopics } = require('../models/models');
+const { fetchTopics, fetchApiDocs } = require('../models/models');
 
-exports.getTopics = (req, res, next) => {
+exports.getTopics = (req, res) => {
     fetchTopics().then(data => {
-        return res.status(200).send({ topics: data });
+        return res.status(200).send({topics: data.rows});
     })
-    .catch(next);
+        .catch(err => { res.status(500).send({ msg: "Internal Server Error" }); });
```

Comment:
This logic should be in error handling middleware. Your catch should just invoke next with the error